### ISSUE TITLE
Components segregation

### ISSUE THREAD

**Issue by Ventan00** [Ventan00/quanda#85]
We have many components and most of them are only used on one system.
How about dividing them into folders according to the system they are used for and put only the shared ones (such as TagSearchBarComponent/PaginationComponent/MessageInputComponent) in the main component folder to make the code more organized?

**Comment by pnarozniak** [comment]
I agree with you to store single-use components inside folder with name according to system they are used in. 
But i think we should store reusable components inside "shared" folder.
And i would also suggest removing "Component" postfix from components name (it makes their name too long, furthermore they are components by default so i think it is unnecessary)

And what do you guys think about dividing pages into folders as well, where the folder name will be part of page path for example (convention used in next.js framework):
> authentication/
>> login (@page "authentication/login")
>> login (@page "authentication/register")
>> ...

> profile/
>> edit (@page="profile/edit")
>> ...

> /
>> home (@page="/home")
>> ...

**Comment by Patrykoss** [comment]
I like idea of dividing components to folders based on page which is used for. And after that we can consider removing postfix "Component" in some of them.

**Comment by Ventan00** [comment]
Ok, Let's do that
